department-of-veterans-affairs/va.gov-cms
# VACMS-8682 re-add SB stuff that was reverted. no performance hit

## Description

Relates to #9102 and #9115. The work added in #8994 added fontawesome support for pseudoelements which I thought may make people's lives easier in the long run, but had a huge performance hit that wasn't noticed on smaller pages. Pages with a ton of DOM elements.... different story. Removing that pseudoelement support, which was not currently used anywhere except for examples in storybook, fixes the performance regression.

This PR adds back the changes from #8994 that were reverted in #9102, minus the pseudoelement support. 

## Testing done
See icons in buttons in storybook, see that pages mentioned in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9115 do not time out.

## Screenshots
<img width="1029" alt="Screen Shot 2022-05-17 at 11 50 41 AM" src="https://user-images.githubusercontent.com/11279744/169405008-8315086c-ecd1-4b6d-9f3a-d8ec6afa2ec1.png">


## QA steps

- Visit storybook buttons page
- Visit drupal modules page
- Visit page that has the monster dropdown select list

Icons should be present in the "icon buttons" and none of the pages should have abnormal performance issues.

### Definition of Done


### Select Team for PR review
